__tests__/app.test.js

```diff
@@ -42,3 +42,43 @@ describe("GET: /api/topics", () => {
       });
   });
 });
+
+describe("GET: /api/articles/:article_id", () => {
+  test("200: responds with article object that matches the given id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        console.log(article);
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          })
+        );
+      });
+  });
+  test("404: sends an appropriate status and error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/999")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid Request");
```

Comment:
I would argue this isn't an invalid request - the request is fine because the data type is correct it just can't find anything. Data/article not found maybe?